addons/mail/static/src/messaging/component/message/message.js

```diff
@@ -583,12 +572,12 @@ Object.assign(Message, {
         hasReplyIcon: false,
         isSelected: false,
         isSquashed: false,
-        threadStringifiedDomain: '[]',
     },
     props: {
         attachmentsDetailsMode: {
-            type: String, //['auto', 'card', 'hover', 'none']
-            optional: true
+            type: String,
+            optional: true,
+            validate: prop => ['auto', 'card', 'hover', 'none'].includes(prop),
```

Comment:
I was wondering and I didn't test, how does validate works with optional? Is it not called if no value is given? Or should it take it into account?

Reply:
At a glance on OWL code, optional and undefined value is validated before, and the validate function is applied. This means a different value than `undefined`, `'auto'`, `'card'`, `'hover'` or `'none'` should crash